### PR TITLE
Skip Julia test

### PR DESCRIPTION
**Pull request by IanMatthewHuff**
For #

-   [ ] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR).
-   [ ] Title summarizes what is changing.
-   [ ] Has a [news entry](https://github.com/Microsoft/vscode-jupyter/tree/main/news) file (remember to thank yourself!).
-   [ ] Appropriate comments and documentation strings in the code.
-   [ ] Has sufficient logging.
-   [ ] Has telemetry for enhancements.
-   [ ] Unit tests & system/integration tests are added/updated.
-   [ ] [Test plan](https://github.com/Microsoft/vscode-jupyter/blob/main/.github/test_plan.md) is updated as appropriate.
-   [ ] [`package-lock.json`](https://github.com/Microsoft/vscode-jupyter/blob/main/package-lock.json) has been regenerated by running `npm install` (if dependencies have changed).

### CODE REVIEW

**Comment by IanMatthewHuff** [comment]
Any other ones needed here?

File: src/test/datascience/notebook/nonPythonKernels.vscode.test.ts [hunk 1]
```diff
@@ -170,6 +170,7 @@ suite('DataScience - VSCode Notebook - Kernels (non-python-kernel) (slow)', () =
         }
     });
     test('Can run a Julia notebook', async function () {
+        this.skip(); // Flakey, tracked by issue 4453
```
Comment (DonJayamanne) [inline src/test/datascience/notebook/nonPythonKernels.vscode.test.ts:173]: I think its better to `return this.skip()`, else rest of the code still executes, and fails silently (as test has been skipped)...

**Review by DonJayamanne — APPROVED** [review]

File: src/test/datascience/notebook/nonPythonKernels.vscode.test.ts (hunk 1 above)
Comment (DonJayamanne) [reply src/test/datascience/notebook/nonPythonKernels.vscode.test.ts:173]: Yeah flaky, failed on me yesterday once...

Comment (IanMatthewHuff) [reply src/test/datascience/notebook/nonPythonKernels.vscode.test.ts:173]: Ahh, didn't know that. In the IDE the code after this.skip looks greyed out as unreachable, but I can change to return this.skip() if it safer. Will do.

**Review by rchiodo — APPROVED** [review]

**Pull request merged**